libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -736,19 +735,22 @@ bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
         check_failed(ARMING_CHECK_PARAMETERS, true, "Mode channel and RC%d_OPTION conflict", rc().flight_mode_channel_number());
         check_passed = false;
     }
-    const RCMapper * rcmap = AP::rcmap();
-    if (rcmap != nullptr) {
+    {
+        auto &xrc = rc();
         if (!rc().option_is_enabled(RC_Channels::Option::ARMING_SKIP_CHECK_RPY)) {
```

Comment:
```suggestion
        if (!xrc.option_is_enabled(RC_Channels::Option::ARMING_SKIP_CHECK_RPY)) {
```
There are a few places still using `rc()`

Reply:
Or use `rc()` everywhere I guess.

Reply:
I did only want to change the lines I needed to.  It would be nice to be consistent, I agree.

Reply:
... I've removed the `xrc` intermediate variable, costing 0 bytes on MatekF405 and 8 on CubeOrange.

Worth it to keep things simple, and I was finding `xrc` ugly, too :-)